### ISSUE TITLE
browserify-istanbul doesn't seem to think my code is executed if i require it through a symlink

### ISSUE THREAD

**Issue by scamden** [devongovett/browserify-istanbul#6]
![screen shot 2014-09-23 at 12 40 30 pm](https://cloud.githubusercontent.com/assets/1128856/4378586/846813b4-4359-11e4-89d1-80c5138609f6.png)

i get red for the execution of the module even though it's required. here's the code for requiring:

util = require('@grid/util');

this works because i have a symlink from node_modules/@grid to the modules directory which contains util. this pattern is very nice for requiring without using brittle paths but istanbul doesn't seem to be able to map it back. any thoughts?

**Comment by dflynn15** [comment]
Was this ever resolved? I am running into a similar problem using the revealing module pattern. For example:

testfile.js:
```js
app.testfile = (function() {

  function mathFunctions() {

    function add(val1, val2) {
      return val1 + val2;
    }

    function minus(val1, val2) {
      return val1- val2;
    }

    return {
      add: add,
      minus: minus
    }
  }

  return {
    math: mathFunctions
  }
})();
```

testspec.js:
```js
'use strict';
global.app = {};
require('../../src/js/testfile.js');
describe('Test file', function() {
var mathFunctions = app.testfile.math();
  describe('Math functions', function() {
    describe('add', function() {
      it('should return correct addition', function() {
        expect(mathFunctions.add(1, 1)).toBe(2);
      });
    });

    describe('minus', function() {
      it('should return correct subtraction', function() {
        expect(mathFunctions.minus(1, 1)).toBe(0);
      });
    });
  })
});
```

but when run my coverage looks like this:
![screenshot 2015-01-15 12 36 40](https://cloud.githubusercontent.com/assets/1205286/5762837/5c1bfe26-9cb3-11e4-9178-c4be289fd086.png)

**Comment by scamden** [comment]
i'm not sure if it was resolved on istanbul's side, but i was able to fox my particular issues by bundling, the files starting from the symlinked directory instead of the other way, just for tests. (i was using browserify). i actually switched to using aliasify (a browserify transform) ultimately because there were just too many issues with symlinks

**Comment by dflynn15** [comment]
Interesting. I'm not sure at all if my issue is the same, however the output seems to be similar enough. I was using `browserify-istanbul` as a transform in Karma and using Karma coverage to generate reports, buuutttt as you can see it is totally not working. :(

**Comment by scamden** [comment]
ya from tha output it looks like it's seeing the execution of the script
only but not the call to mathFunctions.. not sure man sorry

**Comment by devongovett** [comment]
This was fixed by #15 apparently, by upgrading istanbul. I haven't tested though. Please try out v0.2.0 of browserify-istanbul and let me know if it works.
